README.md

```diff
@@ -52,6 +52,8 @@ By default **semantic-release** uses [Angular Commit Message Conventions](https:
 
 Tools such as [commitizen](https://github.com/commitizen/cz-cli) or [commitlint](https://github.com/conventional-changelog/commitlint) can be used to help contributors and enforce valid commit messages.
 
+**semantic-release** does not care about the pull request title, the only thing that matters is the commit message you end up with after a normal merge, squash merge, or rebase. For example, if you squash-merge a pull request, the pull request title will become the main commit message.
```

Comment:
> the pull request title will become the main commit message

only if there is more than 1 commits. If a PR has a single commit, the squash commit message defaults to the message of that commit. 

I don't think we should add this information. It's GitHub specific, `semantic-release` is not

Reply:
> squash commit message defaults to the message of that commit.

Oh, I thought it would always use the PR title in case you only had 1 commit...
But it actually takes the message from the commit instead of the PR title.

> I don't think we should add this information. It's GitHub specific, `semantic-release` is not

Oh well, in that case, feel free to close the PR. 😉